product_docs/docs/biganimal/release/using_cluster/managing_cloud_account/csp_tagging.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: "Tagging AWS resources"
+deepToC: true
+redirects: 
+  - /purl/cloudservice/csp_tagging
+---
+
+Tags are key-value pairs you can apply to cloud provider resources created on your own cloud account, also known as [Your Cloud Account](/edb-postgres-ai/cloud-service/managed/). Resource tagging allows you to have fine-grained access control and granular cost data at the bucket level. When you add a tag, the following resources are labeled: 
```

Comment:
Is it correct to talk about `granular cost data at the bucket level` ? because we tag other resources besides the buckets?

Reply:
```suggestion
Tags are key-value pairs you can apply to cloud provider resources created on your own cloud account, also known as [Your Cloud Account](/edb-postgres-ai/cloud-service/managed/). Resource tagging allows you to have fine-grained audit policy and cost control over your resources. When you add a tag, the following resources are labeled: 
```